routes/apiRoutes.js

```diff
@@ -0,0 +1,69 @@
+const apiRouter = require("express").Router();
+const path = require("path");
+const fs = require("fs");
+
+// uuid for unique id
+const { v4: uuidv4 } = require("uuid");
+const { request } = require("http");
+
+// API GET request
+apiRouter.get("/notes", (req, res) => {
+  // Read db.json file
+  let data = JSON.parse(
+    fs.readFileSync(path.join(__dirname, "../db/db.json"), "utf8")
+  );
+
+  // Send response data of the GET request
+  res.json(data);
+});
+
+// API POST request
+apiRouter.post("/notes", (req, res) => {
+  // new note (request's body)
+  const newNote = req.body;
+
+  // stringify new note
+  console.log(JSON.stringify(newNote));
+
+  // assign unique id to new note
+  newNote.id = uuidv4();
+
+  // read data from db.json
+  let data = JSON.parse(
+    fs.readFileSync(path.join(__dirname, "../db/db.json"), "utf8")
+  );
+  // Push new note in db.json
+  data.push(newNote);
```

Comment:
Nice job using the .push() array method! :-)

Reply:
consider using .unshift() in place of .push() here.
Whilst the push method adds your new note to the end of the array, unshift will add it to the beginning - this will mean that the newest note will always display at the top of the list in the UI :-)